webapi/src/test/scala/org/knora/webapi/slice/ontology/domain/model/CardinalitySpec.scala

```diff
@@ -67,14 +68,48 @@ object CardinalitySpec extends ZIOSpecDefault {
           assertTrue(ExactlyOne.isStricterThan(ZeroOrOne))
         }
       ),
-      suite(s"ExactlyOne $ZeroOrOne")(
+      suite(s"ZeroOrOne $ZeroOrOne")(
```

Comment:
I don't understand the title of this suite. What is meant by "ZeroOrOne $ZeroOrOne"?

Reply:
it is a contained in another `suite` "Cardinality isStricterThan"

so that for this suite the output will become:
```shell
  + Cardinality isStricterThan
    + should fail if String is no IRI
    + should convert an external resourceClassIri to the internal representation
    + AtLeastOne 1-n
      + '1-n' is stricter than 0-1
      + '1-n' is NOT stricter than 1
    + ExactlyOne 1
    + should not convert an already internal iri
      + '1' is stricter than 0-1
      + '1' is stricter than 1-n
    + ZeroOrOne 0-1                <=============== This is the name of the suite above
      + '0-1' is stricter than 1-n
      + '0-1' is NOT stricter than 1
    + Same cardinality is never stricter
    + Unbounded 0-n
      + '1-n' is stricter than 0-n
      + '1 is stricter than 0-n
      + '0-1' is stricter than 0-n
      + '0-n' is NOT stricter than any other
```

Reply:
Cool, that looks nice, I didn't know that.